blueprints/acceptance-test/files/tests/acceptance/__name__-test.js

```diff
@@ -17,14 +17,14 @@ describe('Acceptance: <%= classifiedModuleName %>', function() {
   });
 
   afterEach(function() {
-    Ember.run(application, 'destroy');
+    application.destroy();
   });
 
-  it('can visit /<%= dasherizedModuleName %>', function() {
+  beforeEach(function() {
```

Comment:
There is a proposal in one (or all) of ember-test-helpers, ember-cli, ember-qunit to add `moduleForAcceptance` for exactly the reasons that it chafes.  If you have time, poke around over there and chime in...